hardware/Wireless-Products-Protocols-and-Applications.md

```diff
@@ -1,54 +1,61 @@
-# Wireless Products, Protocols and Applications
-## General Topics Outline
+<details>
+<summary><font size=5>Table of Contents</font> </summary>
 
-  * [Proprietary products and applications](#proprietary-products-and-applications)
-  * [Zigbee/Thread products and applications](#zigbeethread-products-and-applications)
-  * [BLE products and applications](#ble-products-and-applications)
-  * [Z-Wave products and applications](#z-wave-products-and-applications)
+- [1. Proprietary products and applications](#1-proprietary-products-and-applications)
+- [2. Zigbee/Thread products and applications](#2-zigbeethread-products-and-applications)
+- [3. Bluetooth products and applications](#3-bluetooth-products-and-applications)
+- [4. Z-Wave products and applications](#4-z-wave-products-and-applications)
 
-## Proprietary products and applications
+</details>
+
+# 1. Proprietary products and applications
 - Introduction: 
 
   Silicon Labs' proprietary wireless devices provide high-performance wireless connectivity and ultra-low power 32-bit microcontroller options. With support for major frequency bands in the 142 to 1050 MHz range and 2.4 GHz, these devices can be used to develop many wireless communication systems.
 - Chip name conventions:
 
   For proprietary products, there are many FG type chips which are classified with generations, packages, flash size, temperature grades, output power levels, and frequency bands. The general name convention likes:
```

Comment:
Change "name convention likes:" to "name convention is as follows:"

Reply:
Corrected.